Problem1.py

```diff
@@ -0,0 +1,27 @@
+#Time Complexity : O(2^t) where t is the target value
```

Comment:
Check the time complexity, 2^n because we are iterating array. So worst case we have to touch every element.